dll/win32/syssetup/wizard.c

```diff
@@ -2992,6 +2992,22 @@ ProcessSetupInf(
                              (LPBYTE)pSetupData->SourcePath,
                              (wcslen(pSetupData->SourcePath) + 1) * sizeof(WCHAR));
 
+        WCHAR szData[MAX_PATH];
+        LPWSTR pData = szData;
+        ZeroMemory(szData, _countof(szData) * sizeof(WCHAR));
+        GetWindowsDirectoryW(szData, _countof(szData));
+        wcscat(szData, L"\\inf");
+        pData = szData + (wcslen(szData) + 1);
+        wcscpy(pData, pSetupData->SourcePath);
+        pData += (wcslen(pData) + 1) ;
+        *pData = UNICODE_NULL;
```

Comment:
I know the syssetup doesn't seem to use that, but it would be great to use the strsafe functions here...